Spring-2019/linked_lists/SOLUTIONS_2019Spring.md

```diff
@@ -107,21 +88,41 @@ TODO :bug:
 
 TODO :bug:
 
-### 2. PROBLEM 2 TODO :bug:
-
-Source: TODO :bug:
+### 2. Delete Middle Node
 
-#### Naive/Simple Solution
-
-TODO :bug:
+Source: CTCI 6th edition 2.3
 
 #### Optimal Solution
```

Comment:
@tomkb1 it's helpful to have a brief explanation of the reasoning behind the solution in plain english. The code itself looks great otherwise!